admin/src/all.dto.ts

```diff
@@ -103,14 +92,14 @@ export enum UserAuthTypeDto {
 
 export interface AchievementDto {
   id: string;
-  eventId: string;
-  requiredPoints: number;
-  name: string;
-  description: string;
-  imageUrl: string;
-  locationType: AchievementLocationTypeDto;
-  achievementType: AchievementAchievementTypeDto;
-  organizations: string[];
+  eventId?: string;
+  requiredPoints?: number;
+  name?: string;
+  description?: string;
+  imageUrl?: string;
+  locationType?: AchievementLocationTypeDto;
+  achievementType?: AchievementAchievementTypeDto;
```

Comment:
This is a really weird name and I'm pretty sure if AchievementTypeDto is an enum it shouldn't cause this. Maybe rerun updateapi?